drivers/storage/portworx/component/securitycontextconstraints.go

```diff
@@ -217,14 +219,55 @@ func (s *scc) getSCCs(cluster *opcorev1.StorageCluster) []ocp_secv1.SecurityCont
 			Groups:  nil,
 			Users: []string{
 				fmt.Sprintf("system:serviceaccount:%s:%s", cluster.Namespace, pxutil.PortworxServiceAccountName(cluster)),
-				fmt.Sprintf("system:serviceaccount:%s:%s", cluster.Namespace, CSIServiceAccountName),
-				fmt.Sprintf("system:serviceaccount:%s:%s", cluster.Namespace, LhServiceAccountName),
-				fmt.Sprintf("system:serviceaccount:%s:%s", cluster.Namespace, PVCServiceAccountName),
 				fmt.Sprintf("system:serviceaccount:%s:%s", cluster.Namespace, CollectorServiceAccountName),
```

Comment:
seems previously I forgot to add the new telemetry service account `px-telemetry` here, so if it cannot be restricted, can we add that here as well?

Reply:
Hi @pure-jliao what is the difference between px-telemetry and px-metrics-collector

Reply:
Added.